src/components/modal/v2/parts/views/ShortTerm/Content.jsx

```diff
@@ -0,0 +1,72 @@
+/** @jsx h */
+import { h, Fragment } from 'preact';
+import { useState } from 'preact/hooks';
+import Icon from '../../Icon';
+import Instructions from '../../Instructions';
+import Donut from '../../Donut';
+import ProductListLink from '../../ProductListLink';
+import styles from './styles.scss';
+
+import { useServerData } from '../../../lib/providers';
+
+export const ShortTerm = ({ instructions, linkToProductList, disclosure, donutTimestamps, contentBodyRef }) => {
```

Comment:
```suggestion
export const ShortTerm = ({ instructions, linkToProductList, disclosure, qualifying, periodicPayment, donutTimestamps, contentBodyRef }) => {
```
Inside of ```ContentWrapper.jsx``` I suggested we add ```{...useProductMeta(product)}``` to the Short Term component so that we can then import ```qualifying and ```periodicPayment``` information here as props.